src/controllers/projectsController.js

```diff
@@ -0,0 +1,44 @@
+import Project from '../models/project';
+
+const projectsController = (
+  () => {
+    let projectsArray = JSON.parse(localStorage.getItem("projectsArray")) ? JSON.parse(localStorage.getItem("projectsArray")) : [];
+    return {
+      create(name) {
+        if (name === "") name = "Project Name (Please customize name)";
+        let project = new Project(name);
+        // localStorage.setItem(name, JSON.stringify(project));
+        saveToLocalStorage(name, project);
```

Comment:
Where is `saveToLocalStorage` defined? There are no imports to it, so i'm assuming its a global?